api/swagger.yaml

```diff
@@ -1033,7 +1033,7 @@ definitions:
 
   Volume:
     type: "object"
-    required: [Name, Driver, Mountpoint, Labels, Scope, Options]
+    required: [Name, Driver, Mountpoint, Labels, CreatedAt, Scope, Options]
```

Comment:
Just quickly glancing over your PR, wondering if adding a new field, and making it "required" will affect backward compatibility and/or cause issues with upgrading. (didn't look deep into this, so just thinking out loud here 😅